client/src/main/kotlin/io/spine/examples/pingh/client/DesktopClient.kt

```diff
@@ -312,4 +159,24 @@ public class DesktopClient(
             }
             .post()
     }
+
+    /**
+     * Stops observation by provided subscription.
+     */
+    private fun stopObservation(subscription: Subscription) {
+        client.subscriptions()
+            .cancel(subscription)
+    }
+
+    /**
+     * Provides `ClientRequest` on behalf of current client.
+     */
+    private fun clientRequest(): ClientRequest = client.onBehalfOf(userId)
+
+    /**
+     * Closes the client by shutting down the gRPC connection.
```

Comment:
I think, closing the connections is just one of many steps that should be done upon `close()`.

Let's simplify the description.